drivers/spi/spi_context.h

```diff
@@ -206,6 +201,21 @@ static inline void spi_context_cs_control(struct spi_context *ctx, bool on)
 	}
 }
 
+static inline void spi_context_cs_control(struct spi_context *ctx, bool on)
+{
+	_spi_context_cs_control(ctx, on, false);
+}
+
+static inline void spi_context_unlock_unconditionally(struct spi_context *ctx)
+{
+	/* Forcing CS to go to inactive status */
+	_spi_context_cs_control(ctx, false, true);
+
```

Comment:
this causes a regression on stm32 because, stm32 spi driver calls spi_context_unlock_unconditionally at driver init, and there is no yet config attached to the device. cf upper comment.

Reply:
ouch, actually it does that on every drivers for a reason.